src/docs/cli.js

```diff
@@ -0,0 +1,21 @@
+import { execFileSync } from 'child_process';
+import { resolve } from 'path';
+
+const kibanaDir = resolve(__dirname, '..', '..');
+const docsIndexFile = resolve(kibanaDir, 'docs', 'index.asciidoc');
+const docsToolDir = resolve(kibanaDir, '..', 'docs');
```

Comment:
How about an environment variable fallback?  I'm not a big fan of hardcoding alternatives